src/components/matches.component.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import MatchService from '../services/match.service'
+import PlayerService from '../services/player.service'
+import TeamService from '../services/team.service'
+import { Card, Header } from 'semantic-ui-react'
+
+export default class MatchesComponent extends Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      matches: [],
+      players: [],
+      teams: [],
+      team: undefined,
+    }
+  }
+
+  async componentDidMount() {
+    const matches = await MatchService.getMatches(this.props.match.params.leagueId)
+    const players = await PlayerService.getPlayers(this.props.match.params.leagueId)
+    const teams = await TeamService.getTeams(this.props.match.params.leagueId)
```

Comment:
This will do three synchronous calls (it will wait for each other). It can be ran in parallel, with 

const {matches, players, teams} = await Promise.all([
  MatchService.getMatches(this.props.match.params.leagueId),
  PlayerService.getPlayers(this.props.match.params.leagueId),
  TeamService.getTeams(this.props.match.params.leagueId)
])